repository/src/main/java/org/apache/atlas/repository/audit/CassandraBasedAuditRepository.java

```diff
@@ -69,6 +83,17 @@
   public static final String CASSANDRA_PORT_PROPERTY = "atlas.graph.storage.port";
   public static final String CASSANDRA_REPLICATION_FACTOR_PROPERTY = "atlas.EntityAuditRepository.replicationFactor";
   public static final String CASSANDRA_AUDIT_KEYSPACE_PROPERTY = "atlas.EntityAuditRepository.keyspace";
+  
+  //flag to indicate if SSL is enabled for cassandra (server side SSL)
+  public static final String CASSANDRA_SSL_ENABLED = "atlas.graph.storage.sslEnabled";
+  public static final String CASSANDRA_SSL_TRUSTSTORE_LOCATION = "atlas.graph.storage.cassandra.ssl.truststore.location";
+  public static final String CASSANDRA_SSL_TRUSTSTORE_PASSWORD = "atlas.graph.storage.cassandra.ssl.truststore.password";
+
+  // cassandra login credentials
+  public static final String CASSANDRA_USERNAME = "atlas.graph.storage.username";
```

Comment:
The user credentials could use different properties, so you can have different users for audit and atlas keyspaces.